FirstSolution/Part15Task2/Motorcycle.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Part15Task2
+{
+    public class Motorcycle : Auto
+    {
+        private bool hasSidebar;
+
+        public override void printInfo()
+        {
+            Console.WriteLine($"{Brand} auto info:registration number - {RegistrationNum}; " +
+                              $"max speed - {MaxSpeed}; carrying capacity - {CarryingCapacity}; has sidebar - {hasSidebar}");
+        }
+
+        public override int countCarryingCapacity()
+        {
+            if (hasSidebar == false)
+            {
+                return CarryingCapacity = 0;
+            }
+            
+            return base.countCarryingCapacity();
+        }
+
+        public Motorcycle(string brand, string registrationNum, int maxSpeed, int carryingCapacity, bool hasSidebar) 
+            : base(brand, registrationNum, maxSpeed, carryingCapacity)
+        {   
+            this.hasSidebar = hasSidebar;
+            countCarryingCapacity();
+        }
```

Comment:
Constructor should be in front of methods, please change the members order. https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods

Reply:
Sorry I forgot that, and quickly added that block of code. Just changed that.